cmd/apps.go

```diff
@@ -104,11 +106,19 @@ func AppInfo(appID string) error {
 		return err
 	}
 
+	url, err := appURL(s, appID)
+	if checkAPICompatibility(s.Client, err) != nil {
+		if err != errNoDomain {
+			return err
+		}
+		url = errNoDomain.Error()
+	}
+
 	fmt.Printf("=== %s Application\n", app.ID)
 	fmt.Println("updated: ", app.Updated)
 	fmt.Println("uuid:    ", app.UUID)
 	fmt.Println("created: ", app.Created)
-	fmt.Println("url:     ", app.URL)
+	fmt.Println("url:     ", url)
```

Comment:
This isn't from the controller.... it's [from the SDK](https://github.com/deis/controller-sdk-go/blob/master/apps/apps.go#L108), which is making the URL assumption of appname.cluster.com